app/utils/chart-config/versioning.ts

```diff
@@ -1014,23 +1012,29 @@ export const chartConfigMigrations: Migration[] = [
             dimensionIri: v.lineComponentIri,
           });
         }
-      }
 
-      for (const v of Object.values<any>(config.interactiveFiltersConfig)) {
-        if ("componentIri" in v) {
-          v.componentIri = joinIris({
-            unversionedCubeIri,
-            dimensionIri: v.componentIri,
-          });
+        if (newConfig.chartType === "table") {
+          delete newConfig.fields[k];
+          newConfig.fields[joinIris({ unversionedCubeIri, dimensionIri: k })] =
+            v;
         }
+      }
 
+      for (const v of Object.values<any>(
+        newConfig.interactiveFiltersConfig ?? {}
+      )) {
         if ("componentIris" in v) {
           v.componentIris = v.componentIris.map((iri: string) =>
             joinIris({
               unversionedCubeIri,
               dimensionIri: iri,
             })
           );
+        } else if ("componentIri" in v) {
+          v.componentIri = joinIris({
+            unversionedCubeIri,
+            dimensionIri: v.componentIri,
+          });
```

Comment:
I see no tests updated for this commit (11284bf9d0f2ee291653a0a635c5c27a0f9875c6), do we need one ?